src/components/Button/Button.module.scss

```diff
@@ -162,7 +159,6 @@ button,
   padding: var(--Space-16) var(--Space-32);
   @include for-phone-only {
     width: 100%;
-    max-width: 400px;
```

Comment:
Same for here: https://github.com/getethos/ethos-design-system/commit/6eaa934aa41eb4121bde2be7cd1c822115e4e685#diff-f8f2965423d01560e3bd62a4baabb6d1R164

cc / @kenning is this safe to remove? This was ported over from CMS for what's now `.Button.WhiteCTA`